docs/manual/Makefile

```diff
@@ -71,7 +71,7 @@ manual.html: manual.pdf CFLogo.png favicon-checkerframework.png ../api
 	sed -i -e "s%<style type=\"text/css\">%<style type=\"text/css\">\nimg { max-width: 100\%; max-height: 100\%; }%" manual.html
 
 ../api:
-	cd .. && ln -s ../checker/api api
+	cd .. && ln -fs ../checker/api api
```

Comment:
Why is `-f` needed?  This target runs only if the `../api` directory doesn't exist, so there is no need to overwiret existing destination files.